### PR TITLE
Update for 0.18.0

### PR DESCRIPTION
**Pull request by stuhood**

### CODE REVIEW

File: backend/src/main.ts [hunk 1]
```diff
@@ -11,11 +11,11 @@ import {
   CreateResponse,
   ChangesRequest,
   ChangesResponse
-} from "../../api/rbt/thirdparty/prosemirror//v1/authority_rbt.js";
+} from "../../api/reboot_prosemirror//v1/authority_rbt.js";
```
Comment (benh) [inline backend/src/main.ts:14]: Not your bug but can you remove the double `/`?
```suggestion
} from "../../api/reboot_prosemirror/v1/authority_rbt.js";
```

File: api/reboot_prosemirror/v1/authority.proto [hunk 2]
```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package rbt.thirdparty.prosemirror.v1;
+package reboot_prosemirror.v1;
```
Comment (benh) [inline api/reboot_prosemirror/v1/authority.proto:3]: Why the package change? Plan was to move this into Reboot `thirdparty` once we get stability. Can we keep it please? Thanks!

**Review by benh — APPROVED** [review]
Thanks!

**Review by sp33drac3r — APPROVED** [review]

File: api/reboot_prosemirror/v1/authority.proto (hunk 2 above)
Comment (stuhood) [reply api/reboot_prosemirror/v1/authority.proto:3]: There is a fix for the associated issue on the way. I'll wait for `0.18.1` before landing this.

**Pull request closed without merging**